README.md

```diff
@@ -95,3 +95,19 @@ This package depends on [Made.js](https://github.com/Exabyte-io/made.js) and [Wa
 ## Links
 
 1. [Create React App, GitHub Repository](https://github.com/facebook/create-react-app)
+
+
+## CI Docker files
+
+There are two docker files used for testing in CI. In principle, we could use
+more targeted base images for the use case (e.g. `node` or `selenium` images),
+but we want to verify correct behavior
```

Comment:
For the above - should we consider some caching solution(s)? A quick google search yielded:

- https://github.com/marketplace/actions/docker-layer-caching
- https://evilmartians.com/chronicles/build-images-on-github-actions-with-docker-layer-caching

The first option is using DockerHub, which we don't have set up yet.

We can also try using Amazon ECR as @dmitry-spodarets had implemented in https://github.com/Exabyte-io/mat3ra-website-and-cms/blob/latest/.github/workflows/staging.yml.

Note - since the current toolchain is stable and manageable (completes in ~10 min), we might deal with this issue in a separate Jira issue to advance the work.

Reply:
https://exabyte.atlassian.net/browse/SOF-5466